packages/sources/finnhub-secondary/test/integration/adapter-rest.test.ts

```diff
@@ -46,28 +46,41 @@ describe('rest', () => {
       const data = {
         base: 'EUR',
         overrides: {
-          finnhub: { EUR: 'OANDA:EUR_USD' },
+          'finnhub-secondary': { EUR: 'OANDA:EUR_USD' },
```

Comment:
is this still necessary since the overrides were removed?

Reply:
Although we removed the default overrides set on the EA, we do still support overrides being provided in the request. 

I don't think this test is _strictly_ necessary, as we aren't expecting this call pattern today for the feeds this EA supports. However I figured it was safer to have the test so we know if we break this existing functionality, and we can make an intentional choice as to whether to stop supporting it.